source/server/listener_impl.h

```diff
@@ -296,6 +335,19 @@ class ListenerImpl final : public Network::ListenerConfig,
   SystemTime last_updated_;
 
 private:
+  // Helpers for constructor.
+  ListenerImpl* buildAccessLog();
```

Comment:
nit: unless there static I don't see the point in returning `this` in each function. Just have void functions that do their job.